website/static/js/search.js

```diff
@@ -244,6 +244,9 @@ var ViewModel = function(params) {
 
                     self.results.push(result);
                 }
+                if(result.category === 'registration'){
+                    result.registered_date = new $osf.FormattableDate(result.registered_date);
```

Comment:
As per our style guidelines, use `lowerCamelCase` for variable and attribute names.